### PR TITLE
4144 Fix permission and role checking in changes handler and action bar

### PR DESCRIPTION
**Pull request by rmhowe**
# Description

Following on from discussion in the issue, the changes handler has been changed to check for `admin` or `national_admin` roles instead of the `can_access_directly`. The only other use for `can_access_directly` was to show the export action icon which shouldn't be configurable, so this has been modified to check for the same roles.

medic/medic-webapp#4144

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: api/auth.js [hunk 1]
```diff
@@ -66,6 +66,12 @@ module.exports = {
       callback(null, isDbAdmin(userCtx));
     });
   },
+
+  isAdmin: function(userCtx) {
+    return hasRole(userCtx, '_admin') ||
+           hasRole(userCtx, 'national_admin');
+  },
```
Comment (rmhowe) [inline api/auth.js]: This shares most of its functionality with the `Session` service's `isAdmin` function, not sure if worth pulling it out to the shared library/if that's ready for use yet?

File: .gitignore [hunk 2]
```diff
@@ -20,3 +20,4 @@ tests/results
 *.sublime-project
 *.sublime-workspace
 /.env
+/.vscode/
```
Comment (alxndrsn) [inline .gitignore]: Ideally configure a global gitignore for your own editor's specific files.

File: api/auth.js (hunk 1 above)
Comment (alxndrsn) [reply api/auth.js]: Ideally we'd only have one implementation.  But from a quick look, it's not clear to me whether `Session` should be delegating to `Auth` to determine this, or the other way round.

File: .gitignore (hunk 2 above)
Comment (rmhowe) [reply .gitignore]: Ah good idea, will do

File: api/auth.js (hunk 1 above)
Comment (rmhowe) [reply api/auth.js]: Well `Session` is client side and `auth` is server side so I don't think they'd necessarily delegate to each other, just share the same logic in some common utility function in the shared library

**Pull request merged**